crates/vm/src/arch/interpreter.rs

```diff
@@ -42,15 +45,22 @@ pub struct InterpretedInstance<'a, F, Ctx> {
     #[allow(dead_code)]
     pre_compute_buf: AlignedBuf,
     /// Instruction table of function pointers and pointers to the pre-computed buffer. Indexed by
-    /// `pc_index = (pc - pc_base) / DEFAULT_PC_STEP`.
+    /// `pc_index = pc / DEFAULT_PC_STEP`.
+    /// SAFETY: The first `pc_base / DEFAULT_PC_STEP` entries will be unreachable. We do this to
+    /// avoid needing to subtract `pc_base` during runtime.
     pre_compute_insns: Vec<PreComputeInstruction<'a, F, Ctx>>,
```

Comment:
maybe only keep this when `#[cfg(not(feature = "tco"))]`?

Reply:
nevermind it was easy